cmake/Platform/Core/BoardPropertiesReader.cmake

```diff
@@ -127,5 +131,6 @@ function(_try_get_board_property BOARD_ID PROPERTY_NAME OUTPUT_VAR)
             set(VALUE ${${BOARD_NAME}.menu.cpu.${BOARD_CPU}.${PROPERTY_NAME}})
         endif()
     endif()
+    _recursively_replace_properties(${BOARD_ID} VALUE)
```

Comment:
This was already added to `_GET_BOARD_PROPERTY`. Back then I forgot to add it to `_try_get_board_property` as well.